samples/v1.6/Tests/Fallback.Root.Recursive.json

```diff
@@ -0,0 +1,33 @@
+{
+   "type":"AdaptiveCard",
+   "$schema":"http://adaptivecards.io/schemas/adaptive-card.json",
+   "version":"1.6",
+   "fallback":{
+      "type":"TextBlock",
+      "text":"Adaptive Cards not supported"
+   },
+   "body":[
+      {
+         "type":"TextBlock",
+         "text":"This TextBlock should always render",
```

Comment:
This should also not render. Only root fallback should render

Reply:
Yes, You are right. And that works when the CanFallbackToAncestor is set to true. to throw the exception in:

if (bCanFallbackToAncestor && fallbackType != FallbackType::Drop)

Need to verify which components have this flag on true or false.